config/routes.rb

```diff
@@ -0,0 +1,69 @@
+Rails.application.routes.draw do
+  root to: 'home#index'
+
+  get 'home/index' => 'home#index', as: 'homeindex'
+
+  resources :albums
+  resources :books
+  resources :movies
+
+  put "like/:id" => "albums#upvote", as: "upvote"
+  put "like/books/:id" => "books#upvote", as: "bupvote"
+  put "like/movies/:id" => "movies#upvote", as: "mupvote"
```

Comment:
We could actually nest these actions within the resources routes listed above:
```ruby
resources :albums do
  member do
    put 'upvote'
  end
end
```
This gives us the equivalent of:
```ruby
put "/albums/:id/upvote" => "albums#upvote", as: "upvote_album"
```
Which very closely matches the pattern used by the other routes we get with `resources`.